dev/CommonStyles/CommonStyles.vcxitems

```diff
@@ -14,6 +14,11 @@
     <ProjectCapability Include="SourceItemsFromImports" />
   </ItemGroup>
   <ItemGroup>
+    <Page Include="$(MSBuildThisFileDirectory)AppBarSeparator.xaml">
```

Comment:
The file name should follow the `*_themeresources.xaml` convention. I would also move this `<Page>` block to somewhere such as below `AppBarToggleButton` and add the property `<UseNonstandardCondtionalXAML>true</UseNonstandardCondtionalXAML>`

Reply:
Very good point, not sure why I did not do that in the first place; Should be fixed now.